.ci.yaml

```diff
@@ -2752,6 +2752,8 @@ targets:
       tags: >
         ["devicelab","hostonly"]
       task_name: plugin_lint_mac
+      os: Mac-12  # Override OS and Xcode for https://github.com/flutter/flutter/issues/97303
```

Comment:
Is the todo that this would eventually become the default for all the Mac tests?

Reply:
Yes, that works is being done in https://github.com/flutter/flutter/pull/98144.